gapis/api/vulkan/transform_mapping_exporter.go

```diff
@@ -43,20 +43,24 @@ type mappingHandle struct {
 }
 
 type mappingExporter struct {
-	mappings       *map[uint64][]service.VulkanHandleMappingItem
-	thread         uint64
-	path           string
-	traceValues    []mappingHandle
-	notificationID uint64
+	mappings         *map[uint64][]service.VulkanHandleMappingItem
+	thread           uint64
+	path             string
+	traceValues      []mappingHandle
+	notificationID   uint64
+	usedFrameBuffers map[uint64]bool
+	numOfInitialCmds uint64
 }
 
-func newMappingExporter(ctx context.Context, mappings *map[uint64][]service.VulkanHandleMappingItem) *mappingExporter {
+func newMappingExporter(ctx context.Context, numOfInitialCmds uint64, mappings *map[uint64][]service.VulkanHandleMappingItem) *mappingExporter {
 	return &mappingExporter{
-		mappings:       mappings,
-		thread:         0,
-		path:           "",
-		traceValues:    make([]mappingHandle, 0, 0),
-		notificationID: 0,
+		mappings:         mappings,
+		thread:           0,
+		path:             "",
+		traceValues:      make([]mappingHandle, 0, 0),
+		notificationID:   0,
+		usedFrameBuffers: make(map[uint64]bool),
+		numOfInitialCmds: numOfInitialCmds,
```

Comment:
```suggestion
		mappings:         mappings,
		usedFrameBuffers: map[uint64]struct{}{},
		numOfInitialCmds: numOfInitialCmds,
```

- don't init zero value fields
- use nil (zero value) for empty slices
- init maps using the `{}` syntax (for some reason that's what we do in our code base)